### DISCUSSION TITLE
Should add a map editor

### DISCUSSION THREAD

**Discussion by OttoSuwen** [jetelain/ArmaRealMap#54] [Ideas]
I know this is probably a bit of an ask but you should add a map editor so that mistakes can be corrected as yes the map is good but sometimes it can create clipping issues with buildings or the map itself in open map isn't represented properly.

**Comment by jetelain** [comment]
Hi, It's a lot of work, but it would be great.

To correct buildings, and other generation issues, two features are planned:
- Edit map with eden editor, and then integrate changes in the map itself #42
- Export data to Terrain Builder format to allow edit with native tool #41

I think the eden editor will be the most intuitive option for most users, I've already build the mod and processing code (AmsUtil in branch eden-to-wrp), but I still need to integrate them in the Studio user interface.

**Reply by OttoSuwen** [reply]
Wow cant wait then the project is already amazing but with these features, i could probably see it becoming the go-to for all arma 3  map creation
